cudamapper/include/claraparabricks/genomeworks/cudamapper/index.hpp

```diff
@@ -187,6 +187,46 @@ class IndexHostCopyBase
                                                            const cudaStream_t cuda_stream = 0);
 };
 
+/// IndexDescriptorBase - An "almost an Index" class from which an index can be created.
```

Comment:
I would remove this sentence, `IndexDescriptor` only defines which reads go into `Index`, but a `FastaParser` is needed as well in order to create an `Index`

Reply:
removed